modules/widgets/mailchimp.php

```diff
@@ -1,5 +1,11 @@
 <?php
 
+use Automattic\Jetpack\Assets;
+
+use Jetpack_AMP_Support;
```

Comment:
@carlosenamdev 

I'm getting a warning here. Please, check the attached printscreen. It seems that's only visible on non-amp request.
<img width="1305" alt="Screenshot 2020-06-26 at 16 49 16" src="https://user-images.githubusercontent.com/2189187/85876081-182cf680-b7cd-11ea-9870-201fef881a19.png">

Reply:
@pereirinha Thanks for noticing that I removed that `use` code, it working well without it